ChangeLog

```diff
@@ -1,3 +1,18 @@
+Change with release 7.4
+   *) Migration of following addons to support Odoo 8:
+      - l10n_ch_bank
+      - l10n_ch_base_bank
+      - l10n_ch_credit_control_payment_slip_report
+      - l10n_ch_dta
+      - l10n_ch_hr_payroll
+      - l10n_ch_payment_slip
+      - l10n_ch_zip
+    *) All ported addons use the new API.
+    *) Report have been updated when possible to use QwebReport system.
```

Comment:
`Reports have been updated using QwebReport system when possible.`

s to reports if multiple otherwise `Report has been...`